gnucash/register/register-gnome/combocell-gnome.c

```diff
@@ -501,6 +519,62 @@ gnc_combo_cell_set_value (ComboCell *cell, const char *str)
     gnc_basic_cell_set_value (&cell->cell, str);
 }
 
+// This function looks through full_store for a partial match with newval. Note: match_str needs to be freed upon return.
+static void gnc_combo_cell_type_ahead_search(const gchar* newval, GtkListStore* full_store, PopBox *box, char **match_str)
```

Comment:
I think I need to make that new function return a char* rather than use a char** as an input. It will be a lot more in-line with the coding style.